platform-sdk/platform-apps/demos/CryptocurrencyDemo/src/main/java/com/swirlds/demo/crypto/CryptocurrencyDemoState.java

```diff
@@ -227,21 +226,22 @@ public void handleConsensusRound(final Round round, final PlatformState swirldDu
      * {ASK,s,p} = ask to sell 1 share of stock s at p cents (where 1 &lt;= p &lt;= 127)
      * </pre>
      */
-    private void handleTransaction(
-            @NonNull final NodeId id, final boolean isConsensus, @Nullable final Transaction transaction) {
+    private void handleTransaction(@NonNull final NodeId id, @NonNull final Transaction transaction) {
         Objects.requireNonNull(id, "id must not be null");
-        if (transaction == null || transaction.getContents().length == 0) {
+        Objects.requireNonNull(id, "transaction must not be null");
```

Comment:
`s/id/transaction/` in the first argument.

Reply:
fixed